networking/workloads-outside-cluster.md

```diff
@@ -1,49 +1,47 @@
 ---
-title: Configure workload access outside the cluster
-description: Enable and disable internet access for your workloads.
+title: Configure outgoing NAT
+description: Configure Calico Enterprise networking to perform outbound NAT for connections from pods to outside of the cluster, NATing the source IP from the pod IP to the node IP of the node hosting the pod.
 canonical_url: '/networking/workloads-outside-cluster'
 ---
 
 ### Big picture
 
-With all {{site.prodname}} networking options, you can enable and disable internet access for your workloads.
+Configure {{site.prodname}} networking to perform outbound NAT for connections from pods to outside of the cluster, NATing the source IP from the pod IP to the node IP of the node hosting the pod.
```

Comment:
wdyt about making this two sentences? maybe:

> Configure {{site.prodname}} networking to perform outbound NAT for connections from pods to outside of the cluster. {{site.prodname}} optionally source NATs the pod IP to the node IP.

Reply:
Fixed.